example.cfg

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Did you mean to move this file? I don't see anything in the commit description about it, so I don't understand if it is intentional or not.
I am probably fine with moving it to top dir, but then also update the reference in README.md.